docs/api-testing/use-cases/use-drive.md

```diff
@@ -0,0 +1,364 @@
+---
+id: use-drive
+title: Using a File in your Tests
+sidebar_label: Using File from Vault Drive
+description: 'How to use the files from the Drive in your tests'
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The possibilities for how to use the Vault drive in your tests are plenty. This guide shows you some practical scenarios you can face that might require a file inside your tests.
```

Comment:
What type of files are you referring to here? It could be useful to provide some examples in the introduction section.